### PR TITLE
zh-cn(update): sync CORS for the English

### PR DESCRIPTION
**Pull request by fwqaaq**

### CODE REVIEW

File: files/zh-cn/glossary/cors/index.md [hunk 1]
```diff
@@ -3,40 +3,33 @@ title: CORS
...
 
-## 了解更多
-
-### 总体了解
-
-- MDN 上的 [Cross-Origin Resource Sharing (CORS)](/zh-CN/docs/Web/HTTP/CORS)
-- 维基百科上的 [Cross-origin resource sharing](https://zh.wikipedia.org/wiki/Cross-origin_resource_sharing)
-
-### CORS 头
+### CORS 标头
```
Comment (yin1999) [inline files/zh-cn/glossary/cors/index.md:10]: 请同时修改 en-US 文档的标题级别：

```suggestion
## CORS 标头
```

File: files/zh-cn/glossary/cors/index.md [hunk 2]
```diff
@@ -3,40 +3,33 @@ title: CORS
...
+  - : 用于发起一个预检请求，告知服务器正式请求会使用哪一种 [HTTP 请求方法](/zh-CN/docs/Web/HTTP/Methods)。
 - {{HTTPHeader("Origin")}}
-  - : 指示获取资源的请求是从什么域发起的。
+  - : 指示获取资源的请求是从什么源发起的。
 
-### 技术引用
+## 参见
 
+- 在 MDN 的[跨源资源共享（CORS）](/en-US/docs/Web/HTTP/CORS)
+- 在维基百科的[跨源资源共享](https://en.wikipedia.org/wiki/Cross-origin_resource_sharing)
```
Comment (yin1999) [inline files/zh-cn/glossary/cors/index.md:34]:
```suggestion
- 维基百科上的[跨源资源共享](https://zh.wikipedia.org/wiki/跨來源資源共享)
```

**Review by yin1999 — CHANGES_REQUESTED** [review]

**Comment by yin1999** [comment]
请同步修改英文文档：https://github.com/mdn/content/blob/ffb20c5e9a940d083c97758cf73217e638feee1e/files/en-us/glossary/cors/index.md?plain=1#L13

**Comment by fwqaaq** [comment]
update: https://github.com/mdn/content/pull/23427/files

**Review by yin1999 — APPROVED** [review]

**Pull request merged**